core/templates/dev/head/components/StateGraphLayoutService.js

```diff
@@ -148,7 +148,7 @@ oppia.factory('StateGraphLayoutService', [
             if (numBacktrackingCalls <= MAX_BACKTRACKING_CALLS) {
               for (var i = 0; i < adjacencyLists[currentNodeId].length; i++) {
                 if (currentPath.indexOf(
-                      adjacencyLists[currentNodeId][i]) === -1) {
+                  adjacencyLists[currentNodeId][i]) === -1) {
```

Comment:
Would be nice to have custom rule that indents this additionally by 2 (since the if clause spans a continuation line), similar to Python. OK to do this in a separate pass.

Reply:
@seanlip: I tried this out but the present indent rule raises an error if we indent L151 further by 2 spaces. Since consistency is still maintained throughout the codebase, we can adopt these indent settings. How does this sound?
Thanks!

Reply:
If there isn't a way to enforce the "additional indent by 2" (similar to https://legacy.python.org/dev/peps/pep-0008/#indentation for python) then I'm fine with the new convention.

Reply:
Eslint offers [`indent-legacy`](https://eslint.org/docs/rules/indent-legacy) rule but at the same time, their official documentation states: 

>The indent-legacy rule is deprecated and will not receive bugfixes or improvements in the future, so you should eventually switch back to the indent rule.
>To address: We recommend upgrading without changing your indent configuration, and fixing any new indentation errors that appear in your codebase. 

Therefore I think it would be better if we adopted the new convention. 
Thanks!

Reply:
OK, happy to defer to your call. Thanks for the explanation!

Reply:
Great! Let's go ahead with the new convention then. Thanks :)